setup.py

```diff
@@ -64,16 +64,21 @@
     extras_require['all'].extend(reqs)
 
 install_requires = [
-    'Invenio[base,auth,files]{}'.format(invenio_version),
+    'Invenio[base,auth]{}'.format(invenio_version),
+    'invenio-rdm-records>=0.9.0',
+    'invenio-records-permissions>=0.7.0',
+    # Due to search-ui
     'invenio-indexer>=1.1.1,<1.2.0',
     'invenio-jsonschemas>=1.0.1,<1.1.0',
     'invenio-oaiserver>=1.1.1,<1.2.0',
     'invenio-pidstore>=1.1.0,<1.2.0',
     'invenio-records-rest>=1.6.4,<1.7.0',
     'invenio-records-ui>=1.0.1,<1.1.0',
     'invenio-records>=1.3.0,<1.4.0',
-    'invenio-rdm-records>=0.9.0',
-    'invenio-records-permissions>=0.7.0',
+    # Due to files-rest
+    'invenio-files-rest>=1.1.0,<1.2.0',
+    'invenio-iiif>=1.0.0,<1.1.0',
+    'invenio-previewer>=1.1.0,<1.2.0',
```

Comment:
Completely approve. I wouldn't be surprised if we need to keep exploded packages around for the foreseeable future. Overtime maybe even auth will be exploded ;)

Reply:
totally true, its hard to stay with bundles. Maybe when we are more stable we will manage.